src/fluree/db/session.cljc

```diff
@@ -143,7 +143,7 @@
   (swap! (:state session) assoc :db/db (full-load-existing-db session)))
 
 
-(defn indexing?
+(defn indexing-promise-ch
   "Returns block currently being indexed (truthy), or nil (falsey) if not currently indexing."
```

Comment:
As I'm sure you're aware, this gets called in ledger so will need updating over there too.

Should the docstring be updated to reflect that it now returns a promise-chan instead of a block?